docs/Localization.md

```diff
@@ -0,0 +1,45 @@
+# Loop Localization
+
+## Background
+
+Loop (aka Firefox Hello) contains two parts:
+
+* A component that is built into, and shipped with Firefox desktop
```

Comment:
Suggest uising the phrase "an add-on component", since 'add-on' is how it's referred to all over the place.